tests/unit/proc/test_process.py

```diff
@@ -23,14 +23,17 @@ def mock_popen(mocker):
         ["/bin/foo", "-o", "option"],
     ],
 )
-def test_init_args(tmp_dir, args, mocker):
+def test_init_args(tmp_dir, request, args, mocker):
     expected = ["/bin/foo", "-o", "option"]
     proc = ManagedProcess(args)
+    request.addfinalizer(proc._close_fds)
```

Comment:
`ManagedProcess` opens a file for redirecting stdout, which needs to be closed during teardown.